src/main/scala/tilelink/AddressAdjuster.scala

```diff
@@ -7,18 +7,27 @@ import chisel3.util._
 import freechips.rocketchip.config._
 import freechips.rocketchip.diplomacy._
 
-// forceLocal -> used to ensure special devices (like debug) remain reacheable at chip_id=0 even if in params.region
-class AddressAdjuster(val params: ReplicatedRegion, val forceLocal: Seq[AddressSet] = Nil)(implicit p: Parameters) extends LazyModule {
+class AddressAdjuster(
+    val params: ReplicatedRegion, // only devices in this region get adjusted
+    val forceLocal: Seq[AddressSet] = Nil, // ensure special devices (e.g. debug) remain reacheable at id=0 even if in params.region
+    val localBaseAddressDefault: Option[BigInt] = None // default local base address used for reporting manager address metadata
+    )(implicit p: Parameters) extends LazyModule {
   val mask = params.replicationMask
   // Which bits are in the mask?
   val bits = AddressSet.enumerateBits(mask)
   // Find the portion of the addresses which correspond to prefix0
   private def prefix0(region: Seq[AddressSet]): Seq[AddressSet] = {
     region.flatMap { _.intersect(params.local) }
   }
-  private def prefixNot0(region: Seq[AddressSet]): Seq[AddressSet] = {
-    region.flatMap { _.subtract(params.local) }
+  // Default region is used to display address map metadata which corresponds to the expected default choice of prefix value
+  private val defaultRegion: AddressSet = params.local.copy(base = params.local.base + localBaseAddressDefault.getOrElse(0))
```

Comment:
You might consider adding in a require() that the 'localBaseAddressDefault' only uses bits in the replicationMask.
ie: localBaseAddressDefault.foreach { x => require (x & ~mask == 0) }